lib/galaxy/objectstore/s3.py

```diff
@@ -187,7 +189,7 @@ def _initialize(self):
                          'conn_path': self.conn_path}
 
         self._configure_connection()
-        self.bucket = self._get_bucket(self.bucket)
+        self.__bucket = self._get_bucket(self.bucket)
```

Comment:
Is the use of name mangling due to [double leading underscore](https://dbader.org/blog/meaning-of-underscores-in-python) intentional? I think it may break `SwiftObjectStore` which inherits from this class.

Reply:
We are testing `SwiftObjectStore ` so that should be fine. But single underscore will be fine too.